data/data_0DD0A0_3.s

```diff
@@ -5380,13 +5380,13 @@ glabel D_800F0B10
 .asciiz "    th"
 
 .balign 4
-glabel D_800F0B18
+glabel gGrandPrix1stPlacePointReward
```

Comment:
gGrandPrix1stPlacePointReward

How about->
gGrandPrix1stPlaceReward

I'm wondering if GrandPrix should be abreviated as:
gGP...